### PR TITLE
added search and operational info

### PR DESCRIPTION
**Pull request by edewit**
fixes: #2510

### CODE REVIEW

File: src/dashboard/ProviderInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+          </Thead>
+          <Tbody>
+            {providerInfo.map(([name, { providers }]) => (
+              <Tr key={name}>
+                <Td>{name}</Td>
+                <Td>
+                  <ul>
+                    {Object.entries(providers).map(
+                      ([key, { operationalInfo }]) => (
+                        <>
```
Comment (jonkoops) [inline src/dashboard/ProviderInfo.tsx:60]: I believe this Fragment should have a key prop.

File: src/dashboard/ProviderInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                          <li key={key}>{key}</li>
+                          {/* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */}
+                          {operationalInfo && (
```
Comment (jonkoops) [inline src/dashboard/ProviderInfo.tsx:63]: Under what circumstances would `operationalInfo` not be set? Is this type incorrect?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/dashboard/ProviderInfo.tsx (hunk 2 above)
Comment (edewit) [reply src/dashboard/ProviderInfo.tsx:63]: Yes most of the time it's not set

Comment (edewit) [reply src/dashboard/ProviderInfo.tsx:63]: fixed it in PR https://github.com/keycloak/keycloak-nodejs-admin-client/pull/470

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Pull request merged**